health/health.go

```diff
@@ -84,3 +89,11 @@ func writeHealthy(w http.ResponseWriter) {
 type Checker interface {
 	CheckHealth() error
 }
+
+// CheckerFunc implements Checker by calling a function.
```

Comment:
I'd suggest a comment similar to `http.HandlerFunc` to make it more clear what the use case is for this type.

```
The CheckerFunc type is an adapter to allow the use of ordinary functions as Checkers. If f is a function with the appropriate signature, CheckerFunc(f) is a Checker that calls f.
```

Reply:
Done.